src/MuxPort.cpp

```diff
@@ -109,6 +109,28 @@ void MuxPort::handleLinkState(const std::string &linkState)
     )));
 }
 
+//
+// ---> handlePeerLinkState(const std::string &linkState); 
+//
+// handle peer's link state updates
+//
+void MuxPort::handlePeerLinkState(const std::string &linkState)
+{
+    MUXLOGDEBUG(boost::format("port: %s, state db peer link state: %s") % mMuxPortConfig.getPortName() % linkState);
+
+    link_state::LinkState::Label label = link_state::LinkState::Label::Down;
```

Comment:
Are you assuming in all cases that except an up it will be down? In that case we might be doing redundant things in case of an "N/A" which could also be a valid state. 
For Link state down we should exclusively tie it to "down";

Reply:
Agree. I rethink about it, in the case of "N/A", we should assume peer's link state is up, to avoid unnecessary actions as this table is not always up to date. 

Updated accordingly.